### DISCUSSION TITLE
Cable Details with XML File Download

### DISCUSSION THREAD

**Discussion by bmp521** [mattieserver/netbox-topology-views#583] [Ideas]
This is a great plugin. Thank you for creating it. I was wondering if the cable connection details that pop-up when you hover over a cable could be included in the XML download. I would like to download the XML to be opened with Draw.io and be able to see the cable details similar to adding text to a line.

**Comment by dreng** [comment]
First of all, the XML differs from the browser view, because former is built from pure data with our code while the latter is the result of what the physics engine (vis-network) generates. The popup is a function that vis-network provides either. In other words, the XML is not built from what you see on your screen. At the time of writing it is not possible to do what you like.

That said, it would be possible to simulate such a popup by adding a rectangle with text to the XML code. Feel free to add a feature request. This should be much more detailed though. Just adding rectangles to all cables / devices would end in a complete mess pretty soon. So, the question is how to implement such a feature in a good way.

**Comment by snatch2013** [comment]
Yes, this feature would be helpful. How could we add a feature request?

**Reply by snatch2013** [reply]
Ok, I have found it in the issues menu, so I will create a request.

**Comment by snatch2013** [comment]
https://github.com/netbox-community/netbox-topology-views/issues/624
